Gemfile

```diff
@@ -24,13 +24,13 @@ gem 'lograge'
 group :development, :test do
   gem 'envyable', require: 'envyable/rails-now'
   gem 'byebug'
+  gem 'database_cleaner'
```

Comment:
I'm not sure why this was needed to be totally honest - there must be something messed up with the test setup. I'd get undefined constant DatabaseCleaner when it was just in the test group

Reply:
that's weird, i wonder if the specs were being run with `RAILS_ENV=development`?